### ISSUE TITLE
XCode 6.4 freeze

### ISSUE THREAD

**Issue by mikar** [markohlebar/Peckham#46]
Hey,

i've been using this plugin for a while now but with XCode 6.4 this plugin reproducibly freezes XCode on certain projects.
I'm using ~8 plugins and by selectively disabling plugins I found this plugin to be the source of the error.

Sadly, I don't get any error message to give you some hints about whats going on. Just freezes.

**Comment by markohlebar** [comment]
Hey @mikar thanks for reporting this. Couple of bullets
- does it freeze when you open Xcode? 
- can you try it out on a different project and report if that works? 
- does reinstalling the plugin work? 
Cheers!

**Comment by mikar** [comment]
Hey @markohlebar 

This is indeed specific to a certain project which is why I didn't suspect plugins at first but instead cleared DerivedData, ModuleCache, reinstalled XCode, recloned the project etc.

The freeze occurs only if I try to open the project. I had to alt+shift start XCode to get it into a usable state again.

TLDR:
1) no, only when opening the project
2) other projects work though not always, i'll recheck
3) i did a clean install of the plugin after deleting all plugins which did not help

**Comment by markohlebar** [comment]
@mikar that's very unfortunate. Now I'll have to find a project that craps out for me as well. Do you have any open source project that craps out and is on github so I can check against that?

**Comment by mikar** [comment]
I'll try to supply reproducible steps with an openly available project but it might take me a while since I'm at work currently

**Comment by markohlebar** [comment]
Good man, cheers!

**Comment by mikar** [comment]
I have tried with several open-source projects now and cannot reproduce it.
It might just be our unique project of 3.6 GB size and 800k+ LOC. At least, it's the only project where i can reliably reproduce this.

Lets hope its a rare combination, it certainly seems that way ;)

Regards,
mikar

**Comment by markohlebar** [comment]
Hey @mikar, okay I'll just find a project of ridiculous size and try and reproduce this.
